ignite/contrib/metrics/regression/manhattan_distance.py

```diff
@@ -1,13 +1,16 @@
+from typing import Callable, Union
+
 import torch
 
 from ignite.contrib.metrics.regression._base import _BaseRegression
+from ignite.metrics.metric import reinit__is_reduced, sync_all_reduce
 
 
 class ManhattanDistance(_BaseRegression):
     r"""
     Calculates the Manhattan Distance:
 
-    :math:`\text{MD} = \sum_{j=1}^n (A_j - P_j)`,
+    :math:`\text{MD} = \sum_{j=1}^n |A_j - P_j|`,
```

Comment:
I agree that Manhattan distance is computed with abs of error. However, in the paper it is clearly stated without abs. Let's use abs and replace the article citation with sklearn.